DataFormats/METReco/interface/HcalNoiseSummary.h

```diff
@@ -194,6 +195,7 @@ class HcalNoiseSummary
   double rechitEnergy_;
   double rechitEnergy15_;
   double calibCharge_;
+  double lasmonCharge_;
```

Comment:
this variable is not initialized.
Since I've added the MET filter path monitoring a few days ago, now we find that this leads to random differences when reading AOD produced by a release before 100X.
E.g. https://github.com/cms-sw/cmssw/pull/25320#issuecomment-441199764

@perrotta